spec.html

```diff
@@ -35833,7 +35834,7 @@ <h1>
           1. Return the union of _D_ and _C_.
         </emu-alg>
         <emu-note>
-          <p>|ClassRanges| can expand into a single |ClassAtom| and/or ranges of two |ClassAtom| separated by dashes. In the latter case the |ClassRanges| includes all characters between the first |ClassAtom| and the second |ClassAtom|, inclusive; an error occurs if either |ClassAtom| does not represent a single character (for example, if one is \w) or if the first |ClassAtom|'s character value is greater than the second |ClassAtom|'s character value.</p>
+          <p>|ClassRanges| can expand into a single |ClassAtom| and/or ranges of two |ClassAtom| separated by dashes. In the latter case the |ClassRanges| includes all characters in the inclusive interval from the first |ClassAtom| to the second |ClassAtom|; an error occurs if either |ClassAtom| does not represent a single character (for example, if one is \w) or if the first |ClassAtom|'s character value is greater than the second |ClassAtom|'s character value.</p>
```

Comment:
Feels a bit iffy given that we defined intervals to be over numeric types, though I suppose in this part of the spec we did say "character" means "code unit" or "code point" depending on `u` mode.

Reply:
The decision on issue #2181 suggests (to me, at least) that 'code unit' and 'code point' (and thus 'character') should not be treated as numeric types.

Reply:
Yes this is not a good way to describe that set of characters, and uses implicit conversion that we've decided against in #2181. However, I don't think this change makes a difference with that existing problem. I can drop the change until we address #2181, but we'll probably want to use an interval here at that point anyway, so I would keep this change.

Reply:
I think this change makes the problem a bit worse, because the existing notation is not explicitly defined, and the new notation is. Using an imprecisely defined term is better than using a precisely defined term in a place it doesn't apply.

Reply:
Okay I buy that. I'll drop it for now.